client/app/components/group/_activity/_activityDetail/activityDetail.spec.js

```diff
@@ -1,9 +1,9 @@
-import StoreHistoryModule from "./storeHistory";
+import ActivityDetailModule from "./activityDetail";
```

Comment:
Why did you decide to change `storeHistory` to `activityDetail` and not `storeActivity`? I would have found the second one more intuitive. Did I miss something?

Reply:
🙈 I have no idea why I did that. Sorry about that!

Reply:
Actually, looking at this more closely, I think Github might have mistaken where the changes were coming from when I renamed the files.
I believe [this](https://github.com/yunity/foodsaving-frontend/blob/master/client/app/components/group/_history/_historyDetail/historyDetail.spec.js) is the actual file I converted.

Reply:
How confusing! I did a diff of the file trees [1] and all looks good there, and all the file contents look good too. Silly computers :)

[1] https://gist.github.com/nicksellen/1bef6d29edeaf2cb6aabbcffbe86efa0

Reply:
Weird... good to know that stuff like this can happen... 😅